mate-user-guide/C/gospanel.xml

```diff
@@ -49,8 +49,8 @@ these panels.</para>
            
       <variablelist>
       	<varlistentry>
-      		<term><application>Menu Bar</application> applet</term>
-      		<listitem><para>The Menu Bar contains the <guimenu>Applications</guimenu>, <guimenu>Places</guimenu>, and <guimenu>System</guimenu> menus. For more on the menu bar, see <xref linkend="menubar"/>.</para></listitem>
+      		<term><application>Classic Menu</application> applet</term>
+      		<listitem><para>The Classic Menu contains the <guimenu>Applications</guimenu>, <guimenu>Places</guimenu>, and <guimenu>System</guimenu> menus. For more on the classic menu, see <xref linkend="menubar"/>.</para></listitem>
```

Comment:
We should lowercase this for consistency, I think.
`The classic menu contains`

Reply:
I would personally leave it capitalized, as it's the name of the applet. Not _a_ classic menu, but _the_ Classic Menu.

Reply:
The headline of this section specifically refers to the `Classic Menu applet`. The subsequent paragraph explains where to find more information on this `classic menu`. 

I'm looking at this from a translator point of view, where instances of `Classic Menu` is expected to remain as is, while instances of `classic menu` would get localized. My goal is to reduce the use of heavy english words in favor of common terms. The user guide is meant for newcomers, after all.